app/uk/gov/hmrc/eoricommoncomponent/frontend/domain/subscription/NotifyRcmRequest.scala

```diff
@@ -29,6 +30,6 @@ object NotifyRcmRequest {
   implicit val jsonFormat: OFormat[NotifyRcmRequest] = Json.format[NotifyRcmRequest]
 
   def apply(eori: String, name: String, email: String, service: Service): NotifyRcmRequest =
-    new NotifyRcmRequest(dateTimeFormat.format(ZonedDateTime.now()), eori, name, email, service.friendlyName)
+    new NotifyRcmRequest(dateTimeFormat.format(LocalDateTime.now()), eori, name, email, service.friendlyName)
```

Comment:
I don't think it will make any diff to the value but for consistency should should the now() include the ZoneId?
i.e. LocalDateTime.now(ZoneId.of("Europe/London"))

Reply:
Good spot

Reply:
Done